jenkins-job-dsl-test-support/src/main/groovy/com/aoe/gradle/jenkinsjobdsl/JobScriptsSpec.groovy

```diff
@@ -6,6 +6,7 @@ import hudson.model.View
 import javaposse.jobdsl.dsl.*
 import javaposse.jobdsl.plugin.JenkinsJobManagement
 import jenkins.model.Jenkins
+import org.apache.commons.io.FilenameUtils
```

Comment:
You should add apache commons io as dependency if you are using it - or do not rely on it

Reply:
I'm not sure I get that ?